torch/distributed/_functional_collectives.py

```diff
@@ -132,13 +139,15 @@ def unwrap(e: Any):
         out = func(*unwrapped_args, **unwrapped_kwargs)
         return out
 
+
```

Comment:
wanchao are you on a different version of linter than me? why are there so many formatting changes?

prefer a separate pr to update formatting if necessary

Reply:
seems weird as I didn't turn on the auto format on save feature... let me revert those changes

Reply:
probably because I enabled ufmt in my editor and that seems applied autoformatting lol, reverted the format changes back